test/package.json

```diff
@@ -2,22 +2,23 @@
   "name": "bedrock-did-resolver-http-test",
   "version": "0.0.1-0",
   "private": true,
+  "type": "module",
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
-    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
-    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm test",
-    "coverage-report": "nyc report"
+    "coverage": "cross-env NODE_ENV=test c8 --reporter=lcov --reporter=text-summary npm test",
+    "coverage-ci": "cross-env NODE_ENV=test c8 --reporter=lcovonly npm test",
+    "coverage-report": "c8 report"
   },
   "dependencies": {
-    "bedrock": "^4.1.1",
-    "bedrock-express": "^3.2.0",
-    "bedrock-https-agent": "^2.0.0",
+    "@bedrock/core": "^6.0.0",
+    "@bedrock/express": "^8.0.0",
+    "@bedrock/https-agent": "^4.0.0",
+    "@bedrock/mongodb": "^10.0.0",
+    "@bedrock/server": "^5.0.0",
+    "@bedrock/test": "^8.0.0",
     "bedrock-did-resolver-http": "file:..",
-    "bedrock-mongodb": "^8.2.0",
-    "bedrock-server": "^2.7.0",
-    "bedrock-test": "^5.3.2",
-    "cross-env": "^7.0.3",
-    "nyc": "^15.1.0"
+    "c8": "^7.12.0",
+    "cross-env": "^7.0.3"
   },
   "nyc": {
```

Comment:
This should be `c8` as `nyc` is no longer being used.

Reply:
Let's make an issue about this one and update it at a later time. This PR is way beyond crammed with comments right now.

https://github.com/digitalbazaar/bedrock-did-resolver-http/issues/4

Reply:
@aljones15 It's just a small change you need to make. You've already removed `nyc` and added `c8` in this PR. All you need to change is "nyc" on L22 to "c8", and you're good.